### DISCUSSION TITLE
Question About Tests

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2000] [Q&A]
I ran ```forge test``` and all my ```Tests``` passed but when I ran ```forge coverage```, one of them failed. Is that possible?

**Comment by EngrPips** [comment]
Which one failed and run that test that failed alone with the `-vvvv` flag to see if it will fail.

**Reply by Ifra001** [reply]
> Which one failed and run that test that failed alone with the `-vvvv` flag to see if it will fail.

The one that failed once I ran ```forge coverage``` didn't fail when I ran the test alone, so I guess there is no issue?

**Reply by EngrPips** [reply]
I will assume so. Move on. You can run `foundryup` and then retry `forge coverage` to see if  things changed.
